recipes/qt/5.x.x/patches/fix-macdeployqt.diff

```diff
@@ -0,0 +1,21 @@
+diff --git a/src/macdeployqt/macdeployqt/main.cpp b/src/macdeployqt/macdeployqt/main.cpp
+index 0f9164053..5d9e40e36 100644
+--- a/src/macdeployqt/macdeployqt/main.cpp
++++ b/src/macdeployqt/macdeployqt/main.cpp
+@@ -27,6 +27,7 @@
+ ****************************************************************************/
+ #include <QCoreApplication>
+ #include <QDir>
++#include <QLibraryInfo>
+ 
+ #include "../shared/shared.h"
+ 
+@@ -236,7 +237,7 @@ int main(int argc, char **argv)
+     }
+ 
+     if (plugins && !deploymentInfo.qtPath.isEmpty()) {
+-        deploymentInfo.pluginPath = deploymentInfo.qtPath + "/plugins";
++        deploymentInfo.pluginPath = QLibraryInfo::location(QLibraryInfo::PluginsPath);
```

Comment:
From the visible design perspective - I think it'd be better to fill the `pluginPath` from the `shared.cpp`, at the same place as the `qtPath`. 
Then, here you'd need to only check that `!deploymentInfo.pluginPath.isEmpty()`.